crates/core/tedge_mapper/src/main.rs

```diff
@@ -76,15 +78,24 @@ impl fmt::Display for MapperName {
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
     let mapper_opt = MapperOpt::parse();
-    tedge_utils::logging::initialise_tracing_subscriber(mapper_opt.debug);
 
     let component = lookup_component(&mapper_opt.name);
 
     let tedge_config_location =
         tedge_config::TEdgeConfigLocation::from_custom_root(&mapper_opt.config_dir);
     let config = tedge_config::TEdgeConfigRepository::new(tedge_config_location.clone()).load()?;
-    // Run only one instance of a mapper
 
+    let log_level = if mapper_opt.debug {
+        tracing::Level::TRACE
```

Comment:
Buggy because of ` set_log_level()` is buggy and uses the INFO level if the provided level is not DEBUG.

Reply:
Yes, the plan was to have just the `INFO` level, if the log level is not `DEBUG`. This is how it works now, Ruben and I wanted to keep it the same way.